pkg/storage/replica.go

```diff
@@ -170,11 +170,12 @@ const (
 	// proposalIllegalLeaseIndex indicates the proposal failed to apply at
 	// a Lease index it was not legal for. The command should be retried.
 	proposalIllegalLeaseIndex
-	// proposalReproposed indicates the proposal was re-proposed, meaning
-	// it should be retried. However, the original proposal may have succeeded,
-	// so if the retry does not succeed, care must be taken to correctly inform
-	// the caller via an AmbiguousResultError.
-	proposalReproposed
+	// proposalAmbiguousShouldBeReevaluated indicates that it's ambiguous whether
```

Comment:
I think this change is a step backwards. This is not clearer to me than the previous constant's naming and its associated comment.